sops/node-specific/GDI-SOP0002_NCPs-veto-EDIC-decision.md

```diff
@@ -0,0 +1,148 @@
+
+### 7. Summary or Context Diagram
+
+![1+MG DAC, NCP, and Permit Authority processes for approval or veto of a 1+MG recommendation](../../docs/images/GDI-SOP0002_1-summary-diagram.jpg)
+**Figure 1**: 1+MG DAC, NCP, and Permit Authority processes for approval or veto of a 1+MG recommendation 
+
+### 8. Procedure
+#### 1. NCP Distribute the application recommendation
+| Step identifier | When                           | Who       |
+|:----------------|:-------------------------------|:----------|
+| 1               | Receipt of 1+MG DAC decision   | NCP       |
+
+Once the 1+MG DAC issues a decision, as the NCP, you should **distribute the application recommendation** to the appropriate 1+MG Data Holders and Permit Authority(ies), who will approve or deny the recommendation. Remind them to respond within 10 working days.
+
+Move to the following steps depending on the outcome of the step:
+- If a decision is communicated within 10 working days:
+   - If veto is not exercised, let the 1+MG know and move to **step 4**
+   - If **veto is exercised**, move to **step 2**.
+- If a decision is **not** communicated within 10 working days, move to step **step 4**.
+
+#### 2. Communicate justification of veto
+| Step identifier | When                           | Who       |
+|:----------------|:-------------------------------|:----------|
+| 2               | After step 1 and within 1 day  | NCP       |
+
+If the veto of the 1+MG DAC's recommendation is exercised, as the NCP you shall **communicate a detailed justification to the 1+MG DAC within one day**. In this communication, you shall cite discrepancies between the ethical assessment of the 1+MG DAC and the national decision maker(s).
+
+Move to **step 3**.
+
+#### 3. Review of veto by 1+MG DAC
+| Step identifier | When                           | Who       |
+|:----------------|:-------------------------------|:----------|
+| 3               | After step 2                   | 1+MG DAC  |
+
+If veto is exercised on some or all of the relevant data, as the 1+MG DAC, **review the justification provided** by the NCP.
+
+Depending on the outcome of this evaluation:
+- If **veto is accepted**, move to **step 7**. 
+- If the **justification is not accepted** in full, move to **step 6**. 
+- If **no veto was exercised**, or the **time lapse** from step 1 is greater than **10 working days**, move to **step 4**.
+
+#### 4. Inform NCP and User of the decision
```

Comment:
Perhaps "Inform NCP and User **about** the decision"

I find it a bit strange that there will be two decisions communicated to the user (both in step 4 and 7). I wonder if that will be clear to the user. Wouldn't it be enough to communicate the final decision with the user?

Reply:
I believe that steps 4 and 7 are indeed informing the users but of opposite results: approval and rejection of their data requests, so both shouldn't be done simultaneously.